src/wp-admin/js/word-count.js

```diff
@@ -1,10 +1,38 @@
+/**
+ * @namespace wp.utils
+ */
 ( function() {
+	/**
+	 * Word counting object
+	 * 
+	 * @namespace wp.utils.wordcounter
+	 * @memberof wp.utils
+	 *
+	 * @constructs
```

Comment:
Could you elaborate why you decided to use `@constructs` here instead of `@class`?